infrastructure/general/setup-rpc.md

```diff
@@ -1,78 +1,66 @@
-all share the same WebSocket connection at port 9944 on localhost. In this example, you'll set up a
-Polkadot sync node on a Debian-flavored server (such as Ubuntu 22.04). Create a new server on your
-provider of choice or locally at home. See [Set up a Full Node](todo:link){target=_blank} for additional
-instructions. You can install from the default apt repository or build from scratch. The startup
-options in the setup process provide various settings that can be modified.
+Setting up a Polkadot SDK-based node follows a straightforward process. By default, all nodes use port 9944 on localhost for the WebSocket connection. In this guide, you'll set up a Polkadot sync node on a Debian-based server (e.g., Ubuntu 22.04). You can either create a server with your preferred provider or set one up locally. Check out [Set up a Full Node](TODO: add path){target=\_blank} for detailed instructions on installation.
+
+To make the node externally accessible as an archive or pruned RPC node, use the following commands:
 
-A typical setting for an externally accessible Polkadot archive RPC node would be:
+For an externally accessible Polkadot archive RPC node:
 
 ```config
 polkadot --chain polkadot --name myrpc --state-pruning archive --blocks-pruning archive --rpc-max-connections 100 --rpc-cors all --rpc-methods Safe --rpc-port 9944
 ```
 
-Or for a Polkadot pruned RPC node:
+For a Polkadot pruned RPC node:
 
 ```config
 polkadot --chain polkadot --name myrpc --state-pruning 1000 --blocks-pruning archive --rpc-max-connections 100 --rpc-cors all --rpc-methods Safe --rpc-port 9944
 ```
 
-The specified flag options are outlined in greater detail below.
+The options and flags will be explained in the following sections.
 
-### Archive Node vs. Pruned Node
+### Pruned Node vs. Archive Node
 
-A pruned node only keeps a limited number of finalized blocks of the network, not its full history.
-Most frequently required actions can be completed with a pruned node, such as displaying account
-balances, making transfers, setting up session keys, staking, etc. An archive node has the full
-history (database) of the network. It can be queried in various ways, such as providing historical
-information regarding transfers, balance histories, and more advanced queries involving past events.
+A pruned node only keeps a limited number of finalized blocks of the network, not its full history. State and block pruning are two ways of removing old blocks from a system. State pruning removes the states of old blocks while preserving block headers, allowing you to query against headers and hashes but not key-value pairs from the state trie. Block pruning removes the block bodies of old blocks while retaining block headers, allowing you to query against block headers and hashes but not the block body. You can complete many frequently required actions with a pruned node, such as displaying account balances, making transfers, setting up session keys, and staking. 
```

Comment:
I don't know the actual difference between state-pruning and block-pruning, so maybe we want to check with someone more experienced

Reply:
This paragraph is strange. It seems like it says that state pruning and block pruning are different, but then it says that in both of them, you can query against headers and hashes

Reply:
Discussed in Slack, all ok.